zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -434,6 +434,7 @@ impl RusqliteMigration for WalletMigrationAddTxViews {
                    transactions.raw           AS raw,
                    SUM(sent_notes.value)      AS sent_total,
                    COUNT(sent_notes.id_note)  AS sent_note_count,
+                   SUM(CASE WHEN sent_notes.memo IS NULL OR sent_notes.memo = '' THEN 0 ELSE 1 END) AS memo_count,
```

Comment:
How do we store empty memos? Should `memo_count` also exclude the encoding of the empty memo?